ui/templates/base.html

```diff
@@ -58,6 +58,19 @@
         smartlook('init', '{{ APIKEYS.SMARTLOOK }}');
     </script>
     {% endif %}
+    <script type="text/javascript">
```

Comment:
@rhysyngsun please see this thread and https://github.com/mitodl/micromasters/issues/2211#issuecomment-270929044